### PR TITLE
add meeting notes and DOC

### PR DESCRIPTION
**Pull request by sappelhoff**
cc @Remi-Gau I also wrote a bit of documentation, feel free to check. Thanks for establishing this machinery!

### CODE REVIEW

**Comment by Remi-Gau** [comment]
hope it was easy enough to use

**Comment by sappelhoff** [comment]
it was great -- thanks! Feel free to merge when happy.

File: _posts/2023-05-25-Steering-Group-minutes.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+   </td>
+   <td>
+   </td>
+  </tr>
+ </tbody>
+</table>
+
+
+
+[[https://utexas.zoom.us/my/kimray]{.underline}](https://utexas.zoom.us/my/kimray)
```
Comment (Remi-Gau) [inline _posts/2023-05-25-Steering-Group-minutes.md:93]:
```suggestion
```

File: _posts/2023-05-25-Steering-Group-minutes.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr class="even">
+   <td>
+   </td>
+   <td>
+   </td>
```
Comment (Remi-Gau) [inline _posts/2023-05-25-Steering-Group-minutes.md:86]:
```suggestion
```

File: _posts/2023-05-25-Steering-Group-minutes.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+  </tr>
```
Comment (Remi-Gau) [inline _posts/2023-05-25-Steering-Group-minutes.md:87]:
```suggestion
```

File: _posts/2023-06-15-Steering-Group-minutes.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+   </td>
+   <td>
+   </td>
+  </tr>
+ </tbody>
+</table>
+
+
+
+[[https://utexas.zoom.us/my/kimray]{.underline}](https://utexas.zoom.us/my/kimray)
```
Comment (Remi-Gau) [inline _posts/2023-06-15-Steering-Group-minutes.md:139]:
```suggestion
```

File: _posts/2023-07-06-Steering-Group-minutes.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr class="odd">
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
```
Comment (Remi-Gau) [inline _posts/2023-07-06-Steering-Group-minutes.md:113]:
```suggestion
```

File: _posts/2023-07-06-Steering-Group-minutes.md [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+ </tbody>
+</table>
+
+
+
+[[https://utexas.zoom.us/my/kimray]{.underline}](https://utexas.zoom.us/my/kimray)
```
Comment (Remi-Gau) [inline _posts/2023-07-06-Steering-Group-minutes.md:119]:
```suggestion
```

File: _posts/2023-07-27-Steering-Group-minutes.md [hunk 7]
```diff
@@ -0,0 +1,306 @@
...
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr class="odd">
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
```
Comment (Remi-Gau) [inline _posts/2023-07-27-Steering-Group-minutes.md:110]:
```suggestion
```

File: _posts/2023-07-27-Steering-Group-minutes.md [hunk 8]
```diff
@@ -0,0 +1,306 @@
...
+ </tbody>
+</table>
+
+
+
+[[https://utexas.zoom.us/my/kimray]{.underline}](https://utexas.zoom.us/my/kimray)
```
Comment (Remi-Gau) [inline _posts/2023-07-27-Steering-Group-minutes.md:116]:
```suggestion
```

File: _posts/2023-07-27-Steering-Group-minutes.md [hunk 9]
```diff
@@ -0,0 +1,306 @@
...
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr class="odd">
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
```
Comment (Remi-Gau) [inline _posts/2023-07-27-Steering-Group-minutes.md:289]:
```suggestion
```

File: _posts/2023-08-17-Steering-Group-minutes.md [hunk 10]
```diff
@@ -0,0 +1,261 @@
...
+     </li>
+    </ul>
+   </td>
+  </tr>
+ </tbody>
+</table>
+
+
+
+[[https://utexas.zoom.us/my/kimray]{.underline}](https://utexas.zoom.us/my/kimray)
```
Comment (Remi-Gau) [inline _posts/2023-08-17-Steering-Group-minutes.md:189]:
```suggestion
```

File: _posts/2023-09-07-Steering-Group-minutes.md [hunk 11]
```diff
@@ -0,0 +1,239 @@
...
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr class="odd">
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
```
Comment (Remi-Gau) [inline _posts/2023-09-07-Steering-Group-minutes.md:127]:
```suggestion
```

File: _posts/2023-09-07-Steering-Group-minutes.md [hunk 12]
```diff
@@ -0,0 +1,239 @@
...
+ </tbody>
+</table>
+
+
+
+[[https://utexas.zoom.us/my/kimray]{.underline}](https://utexas.zoom.us/my/kimray)
+
```
Comment (Remi-Gau) [inline _posts/2023-09-07-Steering-Group-minutes.md:134]:
```suggestion
```

**Comment by Remi-Gau** [comment]
I have trimmed down some of the tables and will make sure we have no dead links complaints

**Comment by Remi-Gau** [comment]
If I have the energy I may prettify things further but given the traffic those pages I get I am not sure if this is worth the effort...

**Comment by Remi-Gau** [comment]
did some cleaning

**Pull request merged**